source/dofs/dof_tools_constraints.cc

```diff
@@ -2659,9 +2667,9 @@ namespace DoFTools
                         weights[wi][wj] = copy_data.global_parameter_representation[pos](i);
                       }
                   }
-                else
-                  Assert (copy_data.global_parameter_representation[pos](i) == 0,
-                          ExcInternalError());
+              //else
+              //  Assert (copy_data.global_parameter_representation[pos](i) == 0,
+              //          ExcInternalError());
```

Comment:
Yes, it would be better. Commented code is always a bad idea. In 6 months, nobody will remember why the code was commented: is it new code? old code? was it commented by mistake?